src/App.scss

```diff
@@ -24,6 +24,22 @@ html {
 	font-size: 250%;
 }
 
+#slogan {
```

Comment:
Instead of mixing your own css in (did you copy paste that from somewhere in the `App.scss`?) you could try reusing `title-container` class to mitigate the issues with the mobile view ;)

Reply:
I tried specyfing height to `slogan`. Didn't work. That's what you meant right? Because that is the only property of `title-container`. I will play a little bit more with it.